src/gsy_e/gsy_e_core/simulation/results_manager.py

```diff
@@ -0,0 +1,218 @@
+"""
```

Comment:
We have a package called `gsy_e_core.sim_results`. Why don't we put this module (`results_manager`) inside of that package instead? I think it would be a more suitable place for it. What do you think?

Reply:
I do not agree, because the results manager is "only" the caller code of the sim_results modules (amongst others) and belongs into the simulation package. All this code used to be in the `Simulation` class until a few months back.